rc/filetype/sh.kak

```diff
@@ -43,10 +41,30 @@ evaluate-commands %sh{
 
     # Highlight keywords
     printf %s "add-highlighter shared/sh/code/ regex \b($(join "${keywords}" '|'))\b 0:keyword"
+
+}
+
+# For some reasons this needs to be in its own %sh block
```

Comment:
The reason why this "needs" to be in its own %sh block is that this printf from the previous block does not write a newline:
```sh
   printf %s "add-highlighter shared/sh/code/ regex \b($(join "${keywords}" '|'))\b 0:keyword"
```
￼Can you fix this and merge the two %sh blocks please?

Reply:
Thanks for pointing me at the cause of the issue!